web/book/src/tutorial/relations.md

```diff
@@ -16,19 +16,19 @@ looks like this:
 | 5          | 23          | Boston       |       ...       | 13.86 |
 | 6          | 37          | Frankfurt    |       ...       | 0.99  |
 
-A relation is composed of rows, each of which is a tuple containing the same set
-of columns. Each column has an unique name and a designated data type. The table
-above has `invoice_id`, `customer_id`, and `artist_id` columns with a data type
-of "integer number", a `billing_city` column with a data type of "text", a
-number of other columns, and and a `total` column that contains floating-point
+A relation is composed of rows, each of which is a tuple (a list) that contains
+similar columns. Each column has an unique name and a designated data type. The
```

Comment:
```suggestion
A relation is composed of rows. Each row in a relation contains
a value for each of the relation's columns. Each column in a relation has an unique name 
and a designated data type. The
```

I actually don't think we need to mention `tuple` here — we introduce it slightly later